### DISCUSSION TITLE
Error when running Invoke-Monkey365

### DISCUSSION THREAD

**Discussion by LatetoGitHub** [silverhack/monkey365#103] [Q&A]
I'm trying to run Invoke-Monkey365  with the following parameters based on an example in help.

`Invoke-Monkey365 -Instance Microsoft365 -IncludeEntraID  -Analysis Microsoft365  -PromptBehavior SelectAccount`

I get this error:

PropertyNotFoundException: The property 'IsUserInteractive' cannot be found on this object. Verify that the property exists.
InvalidOperation: C:\Users\abhi\Documents\WindowsPowerShell\Modules\monkey365\core\init\Initialize-AuthenticationParam.ps1:40
Line |
  40 |  … lications = [System.Collections.Generic.List[Microsoft.Identity.Clien …
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Unable to find type [System.Collections.Generic.List].

I am using PowerShell version 7.4.2.

**Comment by silverhack** [comment]
It seems that you're import the module incorrectly.

Please, see docs about how to import a PowerShell module.
https://silverhack.github.io/monkey365/install-instructions/
https://learn.microsoft.com/en-us/powershell/scripting/developer/module/importing-a-powershell-module?view=powershell-7.3
